conf/ocsci/enable-multi-storagecluster.yaml

```diff
@@ -0,0 +1,7 @@
+DEPLOYMENT:
+  multi_storagecluster: True
```

Comment:
Can we document the new config values in https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md

mainly:
multi_storagecluster
external_storage_cluster_name
external_storage_cluster_namespace